scan/lib/scan/xcpretty_reporter_options_generator.rb

```diff
@@ -37,6 +41,12 @@ def initialize(open_report, output_types, output_files, output_directory, use_cl
       (@output_types - SUPPORTED_REPORT_TYPES).each do |type|
         UI.error("Couldn't find reporter '#{type}', available #{SUPPORTED_REPORT_TYPES.join(', ')}")
       end
+
+      unless @xcpretty_args.nil?
+        (@xcpretty_args - SUPPORTED_XCARGS_TYPES).each do |type|
```

Comment:
I'm not sure if we actually want to validate these options, TBH 🤔 Validating args like adds a lot of extra support/maintenance when `xcpretty` either adds or removes arguments in the future (see my other comments attached to this review)